v22.2/alter-backup-schedule.md

```diff
@@ -0,0 +1,190 @@
+  814168045421199361 | gcs_backups | ACTIVE | 2022-11-15 18:00:00+00 | @hourly  | BACKUP INTO LATEST IN 'external://gcs_storage' WITH detached
+  814155335856521217 | gcs_backups | ACTIVE | 2022-11-16 00:00:00+00 | @daily   | BACKUP INTO 'external://gcs_storage' WITH detached
+(2 rows)
+~~~
+
+You can use the `SHOW SCHEDULE` statement with one of the schedule IDs to show details for each of the jobs. For the full backup job:
+
+~~~sql
+SHOW SCHEDULE 814155335856521217;
+~~~
+
+This shows that the full backup has a dependent schedule, which lists the incremental backup's schedule ID:
+
+~~~
+          id         |    label    | schedule_status |        next_run        | state | recurrence | jobsrunning | owner |            created            |                                                                                       command
+---------------------+-------------+-----------------+------------------------+-------+------------+-------------+-------+-------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+  814155335856521217 | gcs_backups | ACTIVE          | 2022-11-18 00:00:00+00 | NULL  | @daily     |           0 | root  | 2022-11-15 16:48:10.667783+00 | {"backup_statement": "BACKUP INTO 'external://gcs_storage' WITH detached, "dependent_schedule_id": 814168045421199361}
+(1 row)
+~~~
+
+For the incremental backup:
+
+~~~sql
+SHOW SCHEDULE 814168045421199361;
+~~~
+
+The following includes the `backup_type` as `1`. This signifies that this schedule is for an incremental backup: 
+
+~~~
+          id         |    label    | schedule_status |        next_run        | state | recurrence | jobsrunning | owner |            created            |                                                                       command
+---------------------+-------------+-----------------+------------------------+-------+------------+-------------+-------+-------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------
+  814168045421199361 | gcs_backups | ACTIVE          | 2022-11-15 19:00:00+00 | NULL  | @hourly    |           0 | root  | 2022-11-15 17:52:49.327263+00 | {"backup_statement": "BACKUP INTO LATEST IN 'external://gcs_storage' WITH detached", "backup_type": 1, "dependent_schedule_id": 814155335856521217}
+(1 row)
+~~~
+
+Full backups are implicitly of `backup_type` `0`, and so does not display in the schedule details.
+
+### Apply different options to scheduled backups
+
+You can modify the behavior of your backup schedule and the backup jobs with `SET SCHEDULE OPTION` and `SET WITH`. See the [Schedule options](#schedule-options) table and the [Backup options](#backup-options) table for a list and detail on the available options. 
```

Comment:
nit: `for a list and detail on the available options` --> `for a list of the available options`